revscoring/datasources/__init__.py

```diff
@@ -1,2 +1,65 @@
+"""
+This module implements a set of :class:`~revscoring.datasources.Datasource`
+processors that represent the input data for extracting
+:class:`~revscoring.features.Feature` values.  Just like
+:class:`~revscoring.features.Feature` and other
+:class:'~revscoring.dependencies.Dependent' processors,
```

Comment:
This one looks broken at http://pythonhosted.org/revscoring/revscoring.datasources.html